plugins/analysis-icu/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -1,5 +0,0 @@
-org.elasticsearch.analysis.icu:
```

Comment:
This is unrelated, right? Let's do this in a separate PR. I'm still not sure we should, we could leave this in as we try to re-lock-down the config directory.

Reply:
I did this because this is breaking tests downstream - packaging tests are installing a plugin that has no additional permission so it can happen non-interactively, ant this was the plugin of choice. I can revert this but will need to change more tests downstream, not sure what's better here.